lib/src/components/NxNavigationSidebar/NxNavigationSidebar.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+
+import NxButton from '../NxButton/NxButton';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, propTypes } from './types';
+
+import './NxNavigationSidebar.scss';
+
+const NxNavigationSidebar: FunctionComponent<Props> = function NxNavigationSidebar(props) {
+  const {
+    isOpen,
+    children,
+    className,
+    toggleOpenIcon,
+    toggleCloseIcon,
+    onToggleClick,
+    logoImg,
+    logoText,
+    logoLink
+  } = props;
+
+  const classes = classnames(className, 'nx-page-sidebar', 'nx-page-sidebar--navigation', {
+    'open': isOpen,
+    'closed': !isOpen
+  });
+
+  const toggleButtonIcon = isOpen ? toggleOpenIcon : toggleCloseIcon;
+  const toggleButton = (
+    <NxButton variant="icon-only"
```

Comment:
This has basically none of the normal `.nx-btn` styling.  If you use a plain `<button>` with your custom class you might have to do less styling.

Reply:
> This has basically none of the normal `.nx-btn` styling. If you use a plain `<button>` with your custom class you might have to do less styling.

I tried this and actually ended up adding _more_ styling. So I'm gonna keep using an `NxButton`.